1. Are there any missing values in the data set?
What, if anything should be done about them if there are?

Yes there is missing values
We should replace the missing values with NaN

3. Identify the country with the lowest % of population Under 15 and the one with the highest

In [1]:
import pandas as pd

df=pd.read_csv('data.csv')


High = df['Under15'].max()
low = df['Under15'].min()


print(High)
print(low)


49.99
13.12


The country most populated with Under15 is Niger with 49.99% and least populated with Under15 is Japan with 13.12%

4. Which region has the highest % of population over 60?

In [5]:
import pandas as pd

df=pd.read_csv('data.csv')


High = df['Over60'].max()

print(High)


31.92


The region with the highest population over 60 is Japan with 31.92%



6.Which regions have the lowest literacy rates?

In [4]:
import pandas as pd

df=pd.read_csv('data.csv')


lowest = df['LiteracyRate'].min()

print(lowest)



31.1


In [ ]:
Most African countries have a low literacy rate, with Mali coming out lowest out of all African countries which
include the likes of Lesetho, Malawi,Liberia and so fourth

7.Which regions have the lowest child mortality rates?


In [3]:
import pandas as pd

df=pd.read_csv('data.csv')


lowest = df['ChildMortality'].min()

print(lowest)


2.2


Countries in Europe have the lowest Child Moertality rates with Luxembourg having the lowest rate of 2.2
in amoungst them is Norway and Portugal and other European countries